.circleci/config.yml

```diff
@@ -531,6 +501,15 @@ jobs:
             - /tmp/go-cache
       - store_artifacts:
           path: bin/linux
+      - setup_remote_docker:
+          version: 19.03.13
+          docker_layer_caching: true
```

Comment:
AFAIK this only affects the `machine` executor.

Reply:
I read up a little bit on `docker_layer_caching` (DLC!) and it looks it works with both the `machine` executor and in a `setup_remote_docker`: https://circleci.com/docs/2.0/docker-layer-caching/

There's a point made at that link under *Limitations* which gets at the fact that DLC doesn't working for spinning up a container used as an environment.

I certainly didn't go as far as testing the build time with or without DLC though, so I'm not sure how much difference it actually makes.